scripts/dataset/publish/datagouv/dataset.js

```diff
@@ -0,0 +1,115 @@
+import fsApi from 'fs';
+import path from 'path';
+
+import FormData from 'form-data';
+import nodeFetch from 'node-fetch';
+
+import * as readme from '../../assets/README.template.js';
+import logger from '../../logger/index.js';
+
+const DATASET_LICENSE = 'odc-odbl';
+const DEFAULT_RESOURCE_DESCRIPTION = 'See README.md inside the archive for dataset structure and usage information.';
+
+export async function updateDatasetMetadata({ apiBaseUrl, headers, datasetId, releaseDate, stats }) {
+  const updatePayload = {
+    title: readme.title({ releaseDate }),
+    description: readme.body(stats),
+    license: DATASET_LICENSE,
```

Comment:
Shouldn't the licence be the one defined on data.gouv.fr rather than a constant in the code?

Reply:
I used the license associated with the generated dataset. The one defined on datagouv will be overridden by this one.